docs/functions.rst

```diff
@@ -178,6 +178,12 @@ to stop a database process before restoring files.
    `name`, No, `string`, name of the workload to scale
    `kind`, No, `string`, `deployment` or `statefulset`
    `replicas`, Yes, `int`,  The desired number of replicas
+   `waitForReady`, No, `bool`, Whether to wait for the workload to be ready before executing next steps
+
+.. note::
+   ``waitForReady`` argument is optional and if the argument is not specified
+   in the function, default value is ``true`` which means ``ScaleWorkload``
+   function will always wait for the workload to be in ``Ready`` state by default
```

Comment:
maybe we can just specify the default value in the above table itself. but I will let others have a look.

Reply:
in that case we won't need this paragraph.

Reply:
Agreed. See CreateCSISnapshot and RestoreCSISnapshot documentation for existing examples.

Reply:
Makes sense, I have updated it.